packages/waku/src/lib/plugins/vite-plugin-rsc-transform.ts

```diff
@@ -71,17 +71,26 @@ export ${name === 'default' ? name : `const ${name} =`} createServerReference('$
   }
 };
 
+export const createEmptySpan = (): swc.Span =>
+  ({
+    start: 0,
+    end: 0,
+  }) as swc.Span;
+
 const createIdentifier = (value: string): swc.Identifier => ({
   type: 'Identifier',
   value,
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-expect-error
+  ctxt: 0,
```

Comment:
Hm, we need it in this position... So, if swc fixes the TS type, the `@ts-expect-error` will be gone?

Reply:
yes